test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -292,6 +308,8 @@ BOOST_AUTO_TEST_CASE(events)
 	{
 		"name": "f",
 		"constant": false,
+		"view": false,
```

Comment:
The only case not handled by this is deliberate memory changes via inline assembly.